packages/terra-pills/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "terra-pills",
+  "main": "lib/PillList.js",
+  "version": "0.0.0",
+  "description": "The terra-pills component provides a way to either display a static text or optionally allow trigger actions to launch popovers with the desired content.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git",
+    "directory": "packages/terra-pills"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-pills",
+    "Pills",
+    "Pill List",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "peerDependencies": {
+    "react": "^16.8.5",
+    "react-dom": "^16.8.5",
+    "react-intl": ">=2.8.0 <6.0.0"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "keycode-js": "^2.0.1",
+    "prop-types": "^15.5.8",
+    "terra-theme-context": "^1.0.0",
+    "terra-visually-hidden-text": "^2.32.0"
+  },
+  "scripts": {
+    "compile": "babel --root-mode upward src --out-dir lib --copy-files",
+    "lint": "npm run lint:js && npm run lint:scss",
+    "lint:js": "eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
+    "lint:scss": "stylelint src/**/*.scss",
+    "precompile": "rm -rf lib",
+    "test": "npm run lint && npm run jest && npm run wdio",
+    "jest": "jest --config ../../jestconfig.js",
+    "wdio-default": "cd ../.. && npx wdio",
+    "wdio-lowlight": "cd ../.. && THEME=clinical-lowlight-theme npx wdio",
+    "wdio-fusion": "cd ../.. && THEME=orion-fusion-theme npx wdio",
```

Comment:
you need to update these to work with terra-functional-testing
https://github.com/cerner/terra-core/blob/main/packages/terra-action-footer/package.json#L41-L43